<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTrendAppearance</class>
 <widget class="QWidget" name="QTrendAppearance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_in_out">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>241</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>显示范围</string>
   </property>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>33</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> 最小值：</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_outputMin">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-214748364.000000000000000</double>
    </property>
    <property name="maximum">
     <double>214748364.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>81</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> 最大值：</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_outputMax">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>79</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-214748364.000000000000000</double>
    </property>
    <property name="maximum">
     <double>214748364.000000000000000</double>
    </property>
    <property name="value">
     <double>100.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_pipeSet">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>271</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>通道设置</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>23</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>组名：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_groupNum">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pipeSet">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>通道设置</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pipeCount">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>54</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>通道数：</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_tabType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>521</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>图表类型</string>
   </property>
   <widget class="QRadioButton" name="radioButton_history">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>历史</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_realTime">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>实时</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_showCondition">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>521</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>曲线基本属性</string>
   </property>
   <widget class="QComboBox" name="comboBox_channelNum">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_showAddr">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>LB0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_showAddr_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>103</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>显示条件：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_showCondition">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_showAddr">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>143</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>显示ON/OFF地址：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_showAddr">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../resourse.qrc">
      <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_showAddr_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>63</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>通道号：</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>25</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>所有曲线属性相同</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>241</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>线型设置</string>
    </property>
    <widget class="QLabel" name="label_lineStyle">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>23</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>线型：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_lineStyle">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
      <property name="icon">
       <iconset resource="../../resourse.qrc">
        <normaloff>:/lineType/images/LineType/L1.ico</normaloff>:/lineType/images/LineType/L1.ico</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
      <property name="icon">
       <iconset resource="../../resourse.qrc">
        <normaloff>:/lineType/images/LineType/L3.ico</normaloff>:/lineType/images/LineType/L3.ico</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
      <property name="icon">
       <iconset resource="../../resourse.qrc">
        <normaloff>:/lineType/images/LineType/L2.ico</normaloff>:/lineType/images/LineType/L2.ico</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
      <property name="icon">
       <iconset resource="../../resourse.qrc">
        <normaloff>:/lineType/images/LineType/L4.ico</normaloff>:/lineType/images/LineType/L4.ico</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
      <property name="icon">
       <iconset resource="../../resourse.qrc">
        <normaloff>:/lineType/images/LineType/L5.ico</normaloff>:/lineType/images/LineType/L5.ico</iconset>
      </property>
     </item>
    </widget>
    <widget class="QSpinBox" name="spinBox_lineWidth">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>2</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_lineWidth">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>63</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>线宽：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_showColor">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_showColor">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>101</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>显示颜色：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../resourse.qrc"/>
 </resources>
 <connections/>
</ui>
